scripts/publish.sh

```diff
@@ -28,6 +28,17 @@ fi
 
 npm install
 
+# Utils
+mkdir -p "$GIT_ROOT_DIR/packages/shared/vendor"
+pushd "$GIT_ROOT_DIR/packages/utils"
+npm install --workspaces=false
```

Comment:
with workspaces=false this works with packing the utils package correctly as verified by https://github.com/NangoHQ/nango/actions/runs/8680958308/job/23802673428?pr=1918